lib/ansible/plugins/doc_fragments/rackspace.py

```diff
@@ -1,55 +1,49 @@
-# (c) 2014, Matt Martz <matt@sivel.net>
-#
-# This file is part of Ansible
-#
-# Ansible is free software: you can redistribute it and/or modify
-# it under the terms of the GNU General Public License as published by
-# the Free Software Foundation, either version 3 of the License, or
-# (at your option) any later version.
-#
-# Ansible is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU General Public License for more details.
-#
-# You should have received a copy of the GNU General Public License
-# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2014, Matt Martz <matt@sivel.net>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
 
 
 class ModuleDocFragment(object):
 
     # Standard Rackspace only documentation fragment
-    DOCUMENTATION = """
+    DOCUMENTATION = r'''
 options:
   api_key:
     description:
       - Rackspace API key, overrides I(credentials).
+    type: str
     aliases:
       - password
   credentials:
     description:
       - File to find the Rackspace credentials in. Ignored if I(api_key) and
         I(username) are provided.
+    type: path
     aliases:
       - creds_file
   env:
     description:
       - Environment as configured in I(~/.pyrax.cfg),
         see U(https://github.com/rackspace/pyrax/blob/master/docs/getting_started.md#pyrax-configuration).
+    type: str
     version_added: 1.5
   region:
     description:
       - Region to create an instance in.
+    type: str
     default: DFW
   username:
     description:
       - Rackspace username, overrides I(credentials).
+    type: str
   verify_ssl:
     description:
       - Whether or not to require SSL validation of API endpoints.
+    type: bool
     version_added: 1.5
```

Comment:
quote?

Reply:
~~~diff
- version_added: 1.5
+ version_added: "1.5"
~~~
?

Reply:
Damn, to fix this for all files it's going to be hard to know which PR :-(

Reply:
Fixed.